jobs/build/build-microshift/Jenkinsfile

```diff
@@ -27,8 +27,25 @@ node {
                         string(
                             name: "ASSEMBLY",
                             description: "The name of an assembly to rebase & build for. e.g. 4.9.1",
+                            defaultValue: "test",
                             trim: true
                         ),
+                        string(
+                            name: 'RELEASE_PAYLOADS',
+                            description: '(Optional) Release payloads to rebase against',
```

Comment:
I think we can be clearer here since this could be interpreted as pullspec / release name / nightly name, but if we're just accepting pullspec we can specify here.
And should also mention about arches, since we need specifically [x64, arm], can we say that? Unless in future that list can change/expand?

Reply:
Currently it should be interpreted as pullspec. But I think it would be more convenient to be nightly name.
In any case, i will make it more clear in the description.

Reply:
:+1: